README.md

```diff
@@ -10,6 +10,33 @@ Go Map!! is an iPhone/iPad editor for adding cartographic information to [OpenSt
 Do you want to help testing pre-releases of Go Map!!?
 [Become a TestFlight tester][4] today! 🚀
 
+## Continuous integration
+
+### Prerequisite
+
+- Make sure you have _fastlane_ installed. (From a terminal, change to the `src/iOS` directory and run `bundle install`.)
+- Since _fastlane_ stores your provisioning profiles and certificates in a Git repository (`MATCH_REPO`), you need to create a new, empty repository if you haven't already. The profiles and certificates are protected by a password (`MATCH_PASSWORD`).
+- When creating the Beta locally, _fastlane_ will make sure that your certificates and provisioning profiles are up-to-date.
+
+### How to release a Beta locally
+
+You'll need to obtain the values for the following parameter:
+
+- `MATCH_REPO`: The URL to the Git repository that contains the provisioning profiles/certificates
+- `MATCH_PASSWORD`: The password for encrypting/decrypting the provisioning profiles/certificates
+- `FASTLANE_TEAM_ID`: The ID of the developer team at developer.apple.com
+- `FASTLANE_USER`: The email address that is used to sign in to App Store Connect
+- `FASTLANE_ITC_TEAM_ID`: The ID of the team at appstoreconnect.apple.com
+
+In order to release a new Beta to the TestFlight testers, run
```

Comment:
We could mention that these environment variables could also be placed in `.bash_profile` or `.zshrc`, etc.